content/pages/config.md

```diff
@@ -100,4 +128,46 @@ Token.Toolbar.Arg.Text = 'nobold'
 [favorite_queries]
 ```
 
-All the options are thoroughly documented. 
+## Sample MySQL Option File
+
+```
+[client]
+# The client section is read by mycli and all MySQL applications.
+
+# Default connection information
+user = thor
+password = hammer
+host = localhost
+database = asgard
+port = 3306
+
+# Connect via a socket.
+# socket = '/tmp/mysql.sock'
+
+# Use the UTF-8 character set
+default-character-set = utf-8
+
+# SSL options - see the MySQL documentation for more information.
+# https://dev.mysql.com/doc/refman/5.7/en/secure-connection-options.html#option_general_ssl-ca
+# ssl-ca
+# ssl-cert
+# ssl-key
+# ssl-cipher
+# ssl-verify-server-cert
+
+# Turn on the LOAD DATA INFILE statement
+local-infile = on
+
+# Another local infile alias.
+# Use it if the previous one clashes with other MySQL tools.
+loose-local-infile = on
+
+# Configure the prompt
+prompt = '\h:\d> '
+
+# Configure the pager
+pager = 'vim -'
```

Comment:
We should also stop suggesting the pager thing since this is going to become part of myclirc.